packages/webpack-plugin/src/compiler.ts

```diff
@@ -81,7 +81,7 @@ function compileVanillaSource(
     );
     const ExternalsPlugin = compat.getExternalsPlugin(loader._compiler);
 
-    new NodeTemplatePlugin(outputOptions).apply(childCompiler);
+    new NodeTemplatePlugin().apply(childCompiler);
```

Comment:
`outputOptions` contained just a `filename` property. Looking at both the webpack 4 and 5 docs, this was never a valid property for this plugin. It was causing a type error, so I removed it.